src/data/hyperdrive/convert_data.py

```diff
@@ -280,36 +278,16 @@ def convert_checkpoint_info(checkpoint_info_dict: dict[str, Any]) -> db_schema.C
             if isinstance(value, FixedPoint):
                 value = Decimal(str(value))
         args_dict[key] = value
-    block_checkpoint_info = db_schema.CheckpointInfo(**args_dict)
+    block_checkpoint_info = src.data.hyperdrive.db_schema.CheckpointInfo(**args_dict)
     return block_checkpoint_info
 
 
-def _convert_scaled_value(input_val: int | None) -> Decimal | None:
```

Comment:
We should consider moving this back. This function was meant to be a helper function for converting chain query to postgres specifically, and not a general use function

Reply:
per IRL convo leaving as is since its referrenced in two files